build_containers.sh

```diff
@@ -5,8 +5,9 @@ set -eu -o pipefail
 BUILD_ALL=${BUILD_ALL:=0}
 BUILD_NUMBER=${BUILD_NUMBER:=1}
 TAG=${TAG:-test}
+PUSH=${PUSH:=1}
```

Comment:
I like the idea of decoupling builds from pushing the container images but I'm wondering whether this should be a little more explanatory a name – something like `PUBLISH_CONTAINERS`? — since Git also uses “push” as a term.

Reply:
Good idea. Done